### ISSUE TITLE
Include a changelog for the specification itself

### ISSUE THREAD

**Issue by eemeli** [conventional-commits/conventionalcommits.org#143]
I guess this counts as a bit of a request for dogfooding?

I am using conventional-commits and tools based on it. Currently, when I notice that the website's version has updated itself, I need to look through the commits on GitHub to find out what happened. This is clumsy. What I expected to find was a changelog available somewhere that would summarise the changes for me; this is effectively the most significant reason why I'm encouraging developers around me to use the spec in the first place.

Or did I just not find the changelog, and this is really a request for linking to it more clearly?

**Comment by bcoe** [comment]
@eemeli great point :+1: the cobblers children have no shoes?

Would happily accept a PR adding a CHANGELOG, [standard-verison](https://github.com/conventional-changelog/standard-version) is what I use on all my libraries ... and it's about to have a PR lands that makes it compliant with the latest and greatest rules we're describing in this specification.

**Comment by damianopetrungaro** [comment]
@eemeli do you want to work on it?

**Comment by eemeli** [comment]
@damianopetrungaro I'm afraid I don't really have the bandwidth available for this atm.

**Comment by bcoe** [comment]
@damianopetrungaro I'd be happy to take this on, I think we need to come up with some guidelines for how we tag the different variety of commits to this repo however; as working on a spec document is a bit different than a module ... almost every change could be argued to be `docs`.

**Comment by rbalet** [comment]
Hey @bcoe would you still be open to merge a PR in case I was willing to do one ? 
 I could do one of the both

## A: Conventional changelog
1. use the following library : https://github.com/conventional-changelog/conventional-changelog
2. Add a `release script` inside the `package.json` file 

## B: Release please & GitHub
1. Creating the `release-please.yml` required file
2. Asking you guys to activate the [release-please-action](https://github.com/googleapis/release-please-action) over on the GitHub repo

_Note: never tried release please but seems pretty simple_

Cheers

**Comment by bcoe** [comment]
@rbalet yes I would be open to adding a release process, and would be appreciative of a PR.

**Comment by rbalet** [comment]
@bcoe Nice I'll get to it in the following days. 

Is the actual state ready for you to be the v1.0.0 ?
If yes, could you add a new `v1.0.0` tag in the master branch ? 

Then I'll be able to prepare the changelog with his automation.  
Cheers!
